slushfile.js

```diff
@@ -341,6 +341,26 @@ function configRoxy() {
         '          <path-expression>docFormat</path-expression>\n' +
         '          <range-value-positions>false</range-value-positions>\n' +
         '          <invalid-values>reject</invalid-values>\n' +
+        '        </range-path-index>\n' +
+        '        <range-path-index>\n' +
+        '          <scalar-type>string</scalar-type>\n' +
+        '          <collation>http://marklogic.com/collation/codepoint</collation>\n' +
+        '          <path-expression>eyeColor</path-expression>\n' +
+        '          <range-value-positions>false</range-value-positions>\n' +
+        '          <invalid-values>reject</invalid-values>\n' +
+        '        </range-path-index>\n' +
```

Comment:
Ah wait, we should drop the element index in favor of the path index, and adjust all.xml accordingly, as that works better with the recent xml/json edit features..